src/core/utils/request/onUnhandledRequest.test.ts

```diff
@@ -51,6 +60,27 @@ test('supports the "warn" request strategy', async () => {
   )
 })
 
+test('supports the "warn" request strategy with request body', async () => {
+  await onUnhandledRequest(
+    new Request(new URL('http://localhost/api'), {
+      method: 'POST',
+      credentials: 'include',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      cache: 'no-store',
```

Comment:
Is `cache` and `credentials` crucial to reproduce this behavior? If not, I'd drop them.